README.md

```diff
@@ -30,7 +30,7 @@
 The goal of this challenge is to create a backend application which receives,
 validates and displays data from users.
 
-This repository is designed as a mono-repository for your application and external services. We've prepared an application skeleton for you and assume that your solution will be a light client. In the provided skeleton, please implement [application logic](internal/app/usecase/service.go) which validates and persists [sample user data](assets/data.json) in external service(s), to which your application should connect. Your application should also be able to retrieve this data back and verify its integrity.
+This repository is designed as a mono-repository for your application and external services. We've prepared an application skeleton for you and assume that your solution will be a light client and runs on remote untrusted hosts. In the provided skeleton, please implement [application logic](internal/app/usecase/service.go) which validates and persists [sample user data](assets/data.json) in external service(s), to which your application should connect. Your application should also be able to retrieve this data back and verify its integrity.
```

Comment:
This actually does not make more sense to me.

Reply:
Later in the Slack discussion, you said that’s ok. Isn’t it?

Reply:
Nope " and runs on remote untrusted hosts" just doesn't make sense to me.

Reply:
How would you rephrase it? The point is that yiu can’t store and set to envs any data including private keys.

Reply:
Since it's more about the data, maybe change to something like (if I'm understanding correctly):
```suggestion
This repository is designed as a mono-repository for your application and external services. We've prepared an application skeleton for you and assume that your solution will be a light client that does not store any data on the client itself. In the provided skeleton, please implement [application logic](internal/app/usecase/service.go) which validates and persists [sample user data](assets/data.json) in external service(s), to which your application should connect. Your application should also be able to retrieve this data back and verify its integrity.
```

Reply:
It’s not just about storing. You even can’t pass env variables with a private key. That’s why I think we should just point that the app is running in an untrusted environment.

Reply:
It's up to @QuantumExplorer. Or just merge this and update again later if needed. :man_shrugging: